src/utils/AccessibleButton.ts

```diff
@@ -42,6 +43,11 @@ export class AccessibleButton {
     return this;
   }
 
+  public withoutLabel() {
+    this.hasLabel = false;
+    return this;
+  }
```

Comment:
I preferred introducing this function than removing the error on `ensureCorrectLabel`, since in several cases it's preferable to have a label and this error is useful to avoid forgetting it.